neevaco/neeva-ios
# Add Site Specific Settings

Adds site specific settings UI (based off of Cookie Cutter) behind a flag. 

## Checklists
### How was this tested?

### Manual test cases
- Open URL
- Open a different URL
- Disable tracking protection on URL 1
- Go to URL 2 (verify tracking protection is still on)
- Return to URL 1 (verify tracking protection is still off)

### Screenshots and screen recordings
https://user-images.githubusercontent.com/37356984/158729201-1e677b93-b647-4923-ad40-ae1018d094ca.mp4


## Issues addressed
Fixed: #48 (behind flag)
